components/apimgt/org.wso2.carbon.apimgt.gateway/src/test/java/org/wso2/carbon/apimgt/gateway/handlers/security/APISecurityUtilsTestCase.java

```diff
@@ -66,12 +65,19 @@ public void testSetAuthenticationContext() {
 
         APISecurityUtils.setAuthenticationContext(messageContext, authenticationContext, "abc");
 
+        Assert.assertEquals(APISecurityUtils.getAuthenticationContext(messageContext).getCallerToken(),
+                "callertoken");
+
         Assert.assertEquals("keyType", messageContext.getProperty(APIConstants.API_KEY_TYPE));
 
         //test for IllegalStateException
         String API_AUTH_CONTEXT = "__API_AUTH_CONTEXT";
+        Mockito.when(authenticationContext.getCallerToken()).thenReturn("newCallerToken");
```

Comment:
Shall we use constants here?

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/9250